Sources/xcproj/PBXProj+Helpers.swift

```diff
@@ -36,7 +36,7 @@ extension PBXProj {
     /// - Parameter configReference: reference of the XCBuildConfiguration.
     /// - Returns: config name.
     func configName(configReference: String) -> String? {
-        return (objects.getReference(configReference) as? XCBuildConfiguration)?.name
+        return objects.buildConfigurations[configReference]?.name
```

Comment:
👍  
You've probably done it already, but a search for any of these that are left would be good.